src/components/common/animation-control/speed-control.tsx

```diff
@@ -23,7 +23,7 @@ import styled from 'styled-components';
 import AnimationSpeedSliderFactory from './animation-speed-slider';
 import {Button, CenterFlexbox} from 'components/common/styled-components';
 import {Rocket} from 'components/common/icons';
-import {preciseRound} from 'utils/data-utils';
+import {preciseRound} from '@kepler.gl/layers';
```

Comment:
why is this moved to `layers`? I don't see any layer using it. it is heavily used by components